model_compression_toolkit/gptq/keras/quantization_facade.py

```diff
@@ -47,7 +47,7 @@
 
 
     def get_keras_gptq_config(n_iter: int,
-                              optimizer: OptimizerV2 = tf.keras.optimizers.Adam(),
+                              optimizer: OptimizerV2 = tf.keras.optimizers.Adam(learning_rate=0.2),
```

Comment:
Please use constants.
Also, please update the comment about the default optimizer.